addons/account/models/account_move.py

```diff
@@ -2387,7 +2383,18 @@ def compute_tax_repartition_lines_mapping(move_vals):
 
         move_vals = self.with_context(include_business_fields=True).copy_data(default=default_values)[0]
 
-        tax_repartition_lines_mapping = compute_tax_repartition_lines_mapping(move_vals)
+        is_refund = False
+        if move_vals['move_type'] in ('out_refund', 'in_refund'):
+            is_refund = True
+        elif move_vals['move_type'] == 'entry':
+            base_lines = self.line_ids.filtered(lambda line: line.tax_tag_ids)
```

Comment:
Here, you're filtering the lines with tags (tax_tag_ids=tag_ids in previous versions) ; is it what you want? In the original PR, you were doing line.tax_ids, which should still work here (and makes more sense IMO)

Reply:
>is it what you want? 

Nope. Don't remember changing that. Changed it back.